hawkular-alerts-engine/src/test/java/org/hawkular/commons/cassandra/EmbeddedCassandra.java

```diff
@@ -1,43 +0,0 @@
-/*
```

Comment:
My understanding was that `EmbeddedCassandra` was just wrapping the `EmbeddedCassandraService` from Commons to make the `stop()` method public. I refactored the code using `EmbeddedCassandra` to use `EmbeddedCassandraService` from commons directly. Have I missed some occurrences?